homeassistant/components/zha/const.py

```diff
@@ -102,7 +102,7 @@ def populate_data():
     """
     from zigpy import zcl, quirks
     from zigpy.profiles import PROFILES, zha, zll
-    from homeassistant.components.sensor import zha as sensor_zha
+    from .sensor import RelativeHumiditySensor
```

Comment:
little weird is that Sensor can now no longer fetch any constants. I guess it's a problem for future you ;)

Reply:
Agreed. We’ll clean it up!

Reply:
I think the last commit takes care of this issue... thoughts?